splitio/proxy/router.go

```diff
@@ -72,7 +73,7 @@ func Run(options *ProxyOptions) {
 		api.GET("/splitChanges", splitChanges)
 		api.GET("/segmentChanges/:name", segmentChanges)
 		api.GET("/mySegments/:key", mySegments)
-		api.POST("/testImpressions/bulk", postBulkImpressions)
+		api.POST("/testImpressions/bulk", postImpressionBulk(options.ImpressionListenerEnabled))
```

Comment:
@mredolatti Are you sure that this funcion postImpressionBulk is getting the gin.Context at runtime??